net-libs/libwebsockets/libwebsockets-4.0.1-r1.ebuild

```diff
@@ -12,9 +12,8 @@ SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.
 LICENSE="MIT"
 SLOT="0/16" # libwebsockets.so.16
 KEYWORDS="~amd64 ~arm ~x86"
-IUSE="access-log caps cgi client dbus generic-sessions http-proxy http2 ipv6
-	+lejp libev libevent libressl libuv peer-limits server-status smtp socks5
-	sqlite3 ssl static-libs threads zip"
+IUSE="access-log caps cgi client dbus +dir +file-ops generic-sessions http-proxy +http-uncommon-headers http2 ipv6 +lejp libev libevent libressl libuv peer-limits server-status smtp socks5 sqlite3 ssl static-libs +sys-capability testapps threads zip"
+IUSE+=" ssl_ca_certs ssl_libressl ssl_mbedtls +ssl_openssl"
```

Comment:
Just merge this with the previous IUSE, *however*:

drop ssl_openssl, this is represented by USE="ssl -libressl -mbedtls"

Rename ssl_ca_certs - the name is counter-intuitive, this really should just be the default with no flag to change it, other than via configuration, ie, if a program doesn't explicitly not use the OS CA certs, use the OS installed CA certs.